check_copyright.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# ##################################################
+#  Copyright (c) 2022, Intel Corporation
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
```

Comment:
I suggest using SDPX version of the license, like here: https://github.com/ispc/ispc/blob/0387b0679f2951caeff2a32446790980ac92215b/.github/workflows/rebuild-llvm15.yml

Reply:
fixed